### PR TITLE
Add byo examples for network cidr and api/console ports.

### LINKED ISSUE #1446: hosts.example for byo doesn't document cluster_network_cidr or api_port or console_port

### PR DESCRIPTION
**Pull request by abutcher**
Resolves #1446

### CODE REVIEW

File: inventory/byo/hosts.aep.example [hunk 1]
```diff
@@ -157,6 +157,16 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
...
 
+# Configure SDN cluster network CIDR block. This network block should
+# be a private block and should not conflict with existing network
+# blocks in your infrastructure that pods may require access to.
+# Can not be changed after deployment.
+#osm_cluster_network_cidr=10.1.0.0/16
+
+# Configure master API and console ports.
+#openshift_master_api_port=8443
+#openshift_master_console_port=8443
```
Comment (detiber) [inline inventory/byo/hosts.aep.example]: Not directly related to this, but should we eliminate the separate variables for api_port and console_port, either that or ensure that if api_port is set and console_port is not set that console_port defaults to api_port?

cc @smarterclayton

Comment (liggitt) [inline inventory/byo/hosts.aep.example]: latter... console is separable from api... probably don't want to lose that

**Comment by abutcher** [comment]
@brenton

**Comment by abutcher** [comment]
@brenton should be gtg

@niqhil added an example for `osm_host_subnet_length`

**Pull request merged**